solutions/CSES Distinct Numbers.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: cses-1621
+source: CSES
+title: Distinct Numbers
+author: Andrew Wang
+---
+
+<Spoiler title="Solution">
+
+**Time Complexity:** $\mathcal{O}(N\log{N})$
+
+Sort the array of numbers. Loop through the array and increment the answer for every distinct number. Distinct numbers can be found if the current number isn't equal to the previous number in the array.
```

Comment:
There's already a solution w/ sorting here: https://usaco.guide/bronze/intro-sets?lang=cpp#method-2---sorting-the-array

I think it's fine to just remove the sorting solution from that module.